### PR TITLE
feat: add disintegrate block material

### PR DESCRIPTION
**Pull request by jo-lyn**
image

The idea is for it to look like a charged cube:
- Pulses rapidly when pulled
- Turns red on release

The base stone material should be temporary, will see how the block fits in with the level first!

### CODE REVIEW

File: Assets/ScriptableObjects/Plugin Scripts/DisintegratePlugin.cs [hunk 1]
```diff
@@ -13,10 +13,15 @@ public class DisintegratePlugin : BlockPlugin, IDisplaceable
     private const float Acceleration = 30f;
     private const float GravityAcceleration = 5f;
 
-    private void OnEnable()
+    private Animator _aminator;
```
Comment (nicholaschuayunzhi) [inline Assets/ScriptableObjects/Plugin Scripts/DisintegratePlugin.cs]: there's a typo here 😅

Comment (nicholaschuayunzhi) [reply Assets/ScriptableObjects/Plugin Scripts/DisintegratePlugin.cs]: This also means that animator is required in the main block for disintegrate plugin. I guess we can go with it for now but maybe in the future we can have animators for all blocks or some how do the animation within the plugin itself.

Comment (jo-lyn) [reply Assets/ScriptableObjects/Plugin Scripts/DisintegratePlugin.cs]: good catch 🤣

**Comment by nicholaschuayunzhi** [comment]
hey @jo-lyn, could you update the prefab for disintegrate block? thanks! :)

**Pull request merged**